system-scripts/production_scripts/production_deploy.sh

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -lt 6 ]
+then
+    printf "The script is used like this:\n\n    deploy.sh [ DOMAIN ] [ DB-NAME ] [ DB-USER ]  [ DB-PASSWD ] [ SITE-USER ] [ SITE-USER-EMAIL ] [ENABLE_FILESCAN] [ENABLE_MAILSCAN]\n\n"
+    exit
+fi
+
+domain=$1
+db_name=$2
+db_user=$3
+db_pass=$4
+site_username=$5
+site_useremail=$6
+enable_filescan=$7
+enable_mailscan=$8
+
+secret_key=$(echo `openssl rand -base64 100 | head -c100 | tac`)
```

Comment:
Øhm, brug _altid_ `/dev/urandom` til tilfældige tal.
```suggestion
secret_key=$(dd if=/dev/urandom bs=64 count=1 2> /dev/null | base64 -w 0)
```